async.lua

```diff
@@ -0,0 +1,23 @@
+areas = rawget(_G, "areas") or {}
+
+local safe_file_write = core.safe_file_write
+if safe_file_write == nil then
+    safe_file_write = function(path, content)
```

Comment:
this is dead code. the first revision of MT with async support includes this function.